src/components/search/results/results.vue

```diff
@@ -56,38 +58,63 @@
     PromotedsList,
     ResultsList
   } from '@empathyco/x-components/search';
-  import { defineComponent } from 'vue';
-  import { NextQueriesList } from '@empathyco/x-components/next-queries';
+  import { computed, defineComponent } from 'vue';
+  import { RelatedPromptsList } from '@empathyco/x-components/related-prompts';
   import { useDevice } from '../../../composables/use-device.composable';
   import Result from '../../results/result.vue';
   import { useExperienceControls } from '../../../composables/use-experience-controls.composable';
-  import NextQueryPreview from './custom-next-query-preview.vue';
+  import RelatedPromptsTagList from '../../related-prompts/related-prompts-tag-list.vue';
+  import CustomQueryPreview from './custom-query-preview.vue';
 
   export default defineComponent({
     components: {
       Banner,
       BannersList,
       BaseVariableColumnGrid,
+      CustomQueryPreview,
       MainScrollItem,
-      NextQueryPreview,
-      NextQueriesList,
       Promoted,
       PromotedsList,
+      RelatedPromptsList,
+      RelatedPromptsTagList,
       Result,
       ResultsList
     },
     directives: {
       'infinite-scroll': infiniteScroll
     },
     setup() {
+      const x = use$x();
       const { isMobile } = useDevice();
       const { getControlFromPath } = useExperienceControls();
 
+      const { relatedPrompts, selectedPrompt, selectedQuery } = useState('relatedPrompts', [
+        'relatedPrompts',
+        'selectedPrompt',
+        'selectedQuery'
+      ]);
+
+      const columns = computed(() => (isMobile.value ? 2 : 4));
+
+      const isLowResult = computed(() => 0 < x.totalResults && x.totalResults < 50);
```

Comment:
The lowresults threshold is a config for the semanticQueries module. We could just pick it up from there with ```const { config } = useState('semanticQueries', ['config']);```